src/NuGet.Clients/NuGet.Console/GlobalSuppressions.cs

```diff
@@ -40,3 +40,4 @@
 [assembly: SuppressMessage("Build", "CA1507:Use nameof in place of string literal 'arg'", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetConsole.UtilityMethods.ThrowIfArgumentNullOrEmpty(System.String)")]
 [assembly: SuppressMessage("Build", "CA1806:get_VsTextView creates a new instance of WpfConsoleKeyProcessor which is never used. Pass the instance as an argument to another method, assign the instance to a variable, or remove the object creation if it is unnecessary.", Justification = "<Pending>", Scope = "member", Target = "~P:NuGetConsole.Implementation.Console.WpfConsole.VsTextView")]
 [assembly: SuppressMessage("Build", "CA1501:'ConsoleContainer' has an object hierarchy '9' levels deep within the defining module. If possible, eliminate base classes within the hierarchy to decrease its hierarchy level below '6': 'UserControl, ContentControl, Control, FrameworkElement, UIElement, Visual, DependencyObject, DispatcherObject, Object'", Justification = "<Pending>", Scope = "type", Target = "~T:NuGetConsole.ConsoleContainer")]
+[assembly: SuppressMessage("Maintainability", "CA1507:Use nameof to express symbol names", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetConsole.Implementation.Console.WpfConsole.WriteProgressAsync(System.String,System.Int32)~System.Threading.Tasks.Task")]
```

Comment:
fix the code, don't suppress code analysis warnings.

Reply:
Sure. Fixed problem.